app-arch/bzip2/bzip2-1.0.6-r10.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic multilib-minimal toolchain-funcs
+
+DESCRIPTION="A high-quality data compressor used extensively by Gentoo Linux"
+HOMEPAGE="http://www.bzip.org/"
+SUSE_PV="${PV}.0.2"
+SUSE_P="${PN}-${SUSE_PV}"
+SRC_URI="http://ftp.suse.com/pub/people/sbrabec/bzip2/tarballs/${SUSE_P}.tar.gz"
+S="${WORKDIR}/${SUSE_P}"
+
+LICENSE="BZIP2"
+SLOT="0/1" # subslot = SONAME
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd"
+IUSE="static static-libs"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.0.6-progress.patch
+	"${FILESDIR}"/${PN}-1.0.4-POSIX-shell.patch #193365
+	"${FILESDIR}"/${PN}-1.0.6-mingw.patch #393573
+	"${FILESDIR}"/${PN}-1.0.6-CVE-2016-3189.patch #620466
+)
+
+DOCS=( manual.pdf NEWS README.autotools README README.XML.STUFF )
```

Comment:
Please add CHANGES to DOCS.

Reply:
Fixed.

Reply:
Removed; CHANGES does not exist in the 0.2 suse tarball.

Reply:
In fact, its not present in either of the suse tarballs.